### ISSUE TITLE
Issue with @ApiHeader when its added at the controller level

### ISSUE THREAD

**Issue by sandeepsuvit** [nestjs/swagger#486]
## Current behavior
I tried adding the `@ApiHeader` to my controller and i have two methods in my controller like so

```typescript
@ApiTags('users')
@ApiHeader({ name: 'X-TENANT-ID', description: 'Tenant ID' })
@Controller('users')
export class UsersController {
    constructor(
        private userService: UsersService,
    ) { }

    @Post()
    createUser(@Body() user: CreateUserDto) {
        return this.userService.create(user);
    }

    @Get()
    findAllUsers() {
        return this.userService.findAll();
    }
}
```

## Expected behavior
The expected behavior was the input header should be available for bot the `@Post` and `@Get` methods in the swagger documentation. But it only shows for `@Post`

![Alt text](https://i.ibb.co/svWznYT/screencapture-localhost-4000-apidoc-2020-01-14-14-45-07.png?raw=true "Issue with ApiHeader")

But if i add `@ApiHeader` on top of both `@Post and @Get` it works as expected. Problem happens if i add it at the controller level, as it applies the input field only to the `@Post` method.

## Environment

<pre><code>
Nest version: 6.10.14
 
For Tooling issues:
- Node version: v10.16.0
- Platform:  Mac

Others:
Build works without issues, the field for `X-TENANT-ID` doesn't appear in the `@Get` method
</code></pre>

**Comment by kamilmysliwiec** [comment]
Please, provide a minimal repository which reproduces your issue.

**Comment by sandeepsuvit** [comment]
Hi @kamilmysliwiec here is the minimal repository for the issue stated above `https://github.com/sandeepsuvit/nestjs-swagger-poc`

You can run the application using `npm run start:dev` and access the api doc using this url `http://localhost:3000/apidoc`

If you notice the file `https://github.com/sandeepsuvit/nestjs-swagger-poc/blob/master/src/users/users.service.ts`, i have added the annotation for `@ApiHeader` at the controller level. This is when i get the issue as the header param is a requirement for all the methods. But when i individually add this header annotation for the methods it works.

**Comment by kamilmysliwiec** [comment]
Fixed in 4.1.11. Thanks for reporting!

**Comment by sandeepsuvit** [comment]
Its working now thanks for the quick fix  👍
